config/config.go

```diff
@@ -65,7 +65,7 @@ func get() (*Config, error) {
 		Debug:                          false,
 		EnableCensusTopicSubsection:    false,
 		EnableCustomDataset:            false,
-		EnableFeedbackAPI:              false,
+		EnableFeedbackAPI:              true,
```

Comment:
We typically set all flags to false and use the config in vault/dp-configs to turn it on